presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2906,6 +2906,11 @@ private static TableFinisher createTableFinisher(Session session, TableFinishNod
             else if (target instanceof InsertTarget) {
                 return metadata.finishInsert(session, ((InsertTarget) target).getHandle(), fragments, statistics);
             }
+            else if (target instanceof TableWriterNode.RefreshMaterializedViewTarget) {
+                TableWriterNode.RefreshMaterializedViewTarget refreshMVTarget = (TableWriterNode.RefreshMaterializedViewTarget) target;
```

Comment:
Don't abbreviate names. In this case, it could be renamed to just `refreshTarget` as a concise and to-the-point name.

Reply:
done